lib/matplotlib/pyplot.py

```diff
@@ -450,6 +450,16 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
     clear : bool, default: False
         If True and the figure already exists, then it is cleared.
 
+    tight_layout : bool or dict, default: :rc:`figure.autolayout`
+        If ``False`` use *subplotpars*. If ``True`` adjust subplot
+        parameters using `.tight_layout` with default padding.
+        When providing a dict containing the keys ``pad``, ``w_pad``,
+        ``h_pad``, and ``rect``, the default `.tight_layout` paddings
+        will be overridden.
+
+    **kwargs : optional
+        See `figure.Figure` for other possible arguments.
+
```

Comment:
CI failure is real:  `/home/circleci/project/lib/matplotlib/pyplot.py:docstring of matplotlib.pyplot.figure:46: WARNING: py:obj reference target not found: figure.Figure`